views.py

```diff
@@ -0,0 +1,48 @@
+from flask import Blueprint, jsonify, abort
+from sqlalchemy import desc
+from models import Anime
+
+
+def config_routes(db):
```

Comment:
The main to include the routes in function is to use the function call for app and tests. As each the app.py and conftest.py has different db, including the routes in a function implemented to avoid duplication. Is it sufficient reason or should I try to find a new solution, which might be to switch the import for app or testing case using if/try-catch?